src/OpenTelemetry.PersistentStorage.FileSystem/PersistentStorageHelper.cs

```diff
@@ -160,21 +159,10 @@ internal static string GetUniqueFileName(string extension)
 
     internal static string CreateSubdirectory(string path)
     {
-        string subdirectoryPath = string.Empty;
-        string baseDirectory = string.Empty;
-#if !NETSTANDARD
-        baseDirectory = AppDomain.CurrentDomain.BaseDirectory;
-#else
-        baseDirectory = AppContext.BaseDirectory;
-#endif
-
-        string appIdentity = Environment.UserName + "@" + Path.Combine(baseDirectory, Process.GetCurrentProcess().ProcessName);
-        string subdirectoryName = GetSHA256Hash(appIdentity);
```

Comment:
Just realized that we did not remove the method `GetSHA256Hash` as a part of clean up.

Reply:
Will remove it.